src/DatadogLoggingService.js

```diff
@@ -67,7 +67,8 @@ class DatadogLoggingService extends NewRelicLoggingService {
       trackUserInteractions: true,
       trackResources: true,
       trackLongTasks: true,
-      defaultPrivacyLevel: process.env.DATADOG_PRIVACY_LEVEL || 'mask-user-input',
+      defaultPrivacyLevel: process.env.DATADOG_PRIVACY_LEVEL || 'mask',
+      enablePrivacyForActionName: process.env.DATADOG_ENABLE_PRIVACY_FOR_ACTION_NAME || true,
```

Comment:
Also added `enablePrivacyForActionName: true` here, similar to the changes from this PR: https://github.com/edx/edx-internal/pull/11415